docker-files/docker/app/Dockerfile

```diff
@@ -41,6 +41,7 @@ COPY h5bp /etc/nginx/h5bp
 COPY default /etc/nginx/sites-available/default
 COPY php-fpm.conf /etc/php/7.1/fpm/php-fpm.conf
 COPY xdebug.ini /etc/php/7.1/mods-available/xdebug.ini
+COPY vessel.ini /etc/php/7.1/fpm/conf.d/99-vessel.ini
```

Comment:
Why 99-vessel.ini ...why 99

Reply:
Well, this is loaded on alphabetical order, the default ones comes with `00-` `10-` `20-` and i just added `99-` to be clear it's a custom one there.